src/toil_scripts/gatk_germline/germline.py

```diff
@@ -1,688 +1,357 @@
 #!/usr/bin/env python2.7
-
-""" 
-GATK HaplotypeCaller Genotyping Mode
-
-            Tree Structure of GATK Pipeline
-            0 --> 1 --> 2 --> 3 --> 4
-                                    |
-                                    5
-                                   / \
-                                  6   7
-                                 /     \
-                                8       9
-0 = Start Node
-1 = Download Reference
-2 = Index Reference
-3 = Reference Dictionary
-4 = Index Samples
-5 = HaplotypeCaller SNP & Indel
-6 = VariantRecalibrator SNPs
-7 = VariantRecalibrator Indels
-8 = ApplyRecalibration SNPs
-9 = ApplyRecalibration Indels
-
-===================================================================
-:Dependencies:
-curl            - apt-get install curl
-docker          - apt-get install docker (or 'docker.io' for linux)
-toil            - pip install --pre toil
-"""
 from __future__ import print_function
-
 import argparse
-import errno
 import multiprocessing
 import os
-import shutil
-import subprocess
-import sys
-import textwrap
-from collections import OrderedDict
-
 import yaml
+import synapseclient
+
+from copy import deepcopy
+from urlparse import urlparse
+from bd2k.util.processes import which
+from germline_config import generate_config, generate_manifest
```

Comment:
Imports need to be refactored for PEP8 compliance and consistency with the rest of toil_scripts:

* Order:
  * Python core
  * External imports
  * Imports from same project
* Separate each category with newlines
* Within a category, sort with alpha order